### PR TITLE
Add Hack Night documentation

### PR DESCRIPTION
**Pull request by maxwofford**
@ssun098 I don't have write access to this repo. Would you please take point on reviewing this PR?

### CODE REVIEW

**Comment by selynna** [comment]
@MaxWofford could you add a link to Hack Night in the base folder's README.md?

File: hack_night/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Hack Night
+
+**Table of Contents**
+
+- [Hack Night](#hack-night)
+  - [Preperation](#preperation)
```
Comment (selynna) [inline hack_night/README.md]: Preparation*

File: hack_night/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    - [Demos](#demos)
+
+## Preperation 
+
+Here's a checklist of things that needs to be done before a Hack Night:
+
+- [ ] Choose a time and date
+- [ ] Choose a theme
+- [ ] Announce Hack Night
+  - [ ] Facebook
```
Comment (selynna) [inline hack_night/README.md]: Not sure if we should do FB

File: hack_night/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  - [ ] Email
+  - [ ] Twitter
```
Comment (selynna) [inline hack_night/README.md]: Same with Twitter

File: hack_night/README.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  - [ ] [Slack](https://starthackclub.slack.com)
+    - [ ] [#annoucements](https://starthackclub.slack.com/messages/annoucements)
+    - [ ] [#hack-night](https://starthackclub.slack.com/messages/hack-night)
+
+## Event Schedule
+
+### Introductions
+
+Introductions are an icebreaker activity to help hackers learn eachother's names.
```
Comment (selynna) [inline hack_night/README.md]: each other's*

File: hack_night/README.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+1. [ ] Announce theme
+2. [ ] All projects should have a permalink (ex. use USERNAME.github.io instead of c9.io)
+3. [ ] Check-ins will occur throughout the event
+4. [ ] Event schedule (when will demos occur?)
+
+### Check-ins
+
+Check-ins are an optional way for organizers to see where everyone is at in with their projects.
+
+_**Hackers may have muted their chat when hacking. Make sure to ping the [#hack-night](https://starthackclub.slack.com/messages/hack-night) channel before getting check-ins started.**_
```
Comment (selynna) [inline hack_night/README.md]: maybe add something about using `@here` to ping everyone

File: hack_night/README.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+The intro talk is given by an organizer of the Hack Night. Make sure to cover the following topics:
+
+1. [ ] Announce theme
+2. [ ] All projects should have a permalink (ex. use USERNAME.github.io instead of c9.io)
+3. [ ] Check-ins will occur throughout the event
+4. [ ] Event schedule (when will demos occur?)
+
+### Check-ins
+
+Check-ins are an optional way for organizers to see where everyone is at in with their projects.
```
Comment (selynna) [inline hack_night/README.md]: Optional for organizers to run, or hackers to participate in?

**Comment by maxwofford** [comment]
@ssun098 Addressed!

**Pull request merged**